Habitica/res/values/strings.xml

```diff
@@ -368,7 +369,7 @@ To start, which parts of your life do you want to improve?</string>
     <string name="share_hatched" formatted="false">I just hatched a %1$s %2$s pet in Habitica by completing my real-life tasks!</string>
     <string name="share_raised" formatted="false">I just gained a %1$s %2$s mount in Habitica by completing my real-life tasks!</string>
     <string name="open_in_store">Open in play store</string>
-    <string name="change_class_confirmation">Are you sure you want to change your class? This will cost 3 gems.</string>
+    <string name="change_class_confirmation">Are you sure you want to change your class? This will cost 3 gems</string>
```

Comment:
Okay ! 
I also added the warning about the gear issue
![warning change class](https://cloud.githubusercontent.com/assets/11226450/20907385/33e5115c-bb4e-11e6-9a99-ed715f4c3d6c.PNG)